darts/models/forecasting/transformer_model.py

```diff
@@ -8,6 +8,8 @@
 
 import torch
 import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn import Transformer
```

Comment:
maybe import just `generate_square_subsequent_mask`

Reply:
After reading up a little and checking out the implementation, it turns out that `generate_square_subsequent_mask` is a static method of `Transformer.` While it is possible to import just it (https://stackoverflow.com/questions/48178011/import-static-method-of-a-class-without-importing-the-whole-class) I don't think it's worth it. That said, I definitely agree that this import is a little intuitive and I think that a nice middle ground would be adding an implementation of `generate_square_subsequent_mask` to darts/utils as it's a very small function. What do you think?

Reply:
WDYT @dennisbader ?